thorlcr/master/thmastermain.cpp

```diff
@@ -967,6 +967,23 @@ int main( int argc, const char *argv[]  )
                 throwStringExceptionV(0, "Failed to connect to all nodes");
             PROGLOG("verified mp connection to rest of cluster");
 
+#ifdef _CONTAINERIZED
+            if (globals->getPropBool("@_dafsStorage"))
```

Comment:
@jakesmith 
This will not work if there are multiple replicas for the same component.  Should probably document that, and make it clear this is a developer only debugging option.

Reply:
yes, for that to work, when there are multiple instances of Thor, the SP name/group name, that each is using, needs to be unique.
I hadn't considered that scenario, but it's an easy change and makes it a bit more flexible, so I'll make that change.

Reply:
I will add some notes in the code, describing what, how and why this option is there.